iaso/api/groups.py

```diff
@@ -164,12 +164,23 @@ def dropdown(self, request, *args):
         if user and user.is_authenticated:
             account = user.iaso_profile.account
             # Filter on version ids (linked to the account)""
-            versions = SourceVersion.objects.filter(data_source__projects__account=account)
+            default_version = (
+                self.request.query_params.get("defaultVersion")
+                if self.request.query_params.get("defaultVersion")
+                else account.default_version.id
+            )
+            versions = SourceVersion.objects.filter(data_source__projects__account=account, pk=default_version)
 
         else:
             # this check if project need auth
+            default_version = (
+                self.request.query_params.get("defaultVersion")
+                if self.request.query_params.get("defaultVersion")
+                else project.account.default_version.id
```

Comment:
> Unresolved reference 'project'

This will not work and raise an error 500 because `project` is defined after this line.

Reply:
Thank you for the remark.